### PR TITLE
refactor: errors display

### PR DESCRIPTION
**Pull request by asbiin**
You fool, don't forget these steps:

- [ ] Unit tests
- [ ] Tests with Cypress
- [ ] Documentation
- [ ] Dummy data

### CODE REVIEW

File: resources/js/Shared/Errors.vue [hunk 1]
```diff
@@ -8,18 +8,38 @@
...
 <template>
   <div>
-    <div v-if="errors.length > 0" class="border-red ba br3 pa3" :class="classes">
+    <div v-if="dataerror || exception" class="border-red ba br3 pa3" :class="classes">
       <p class="mv0 fw6">{{ $t('app.error_title') }}</p>
-      <p class="mb0">{{ errors[0] }}</p>
-      <template v-if="errors[1]">
-        <p v-for="item in errors[1]" :key="item.index" class="mb0 mt2">{{ item[0] }}</p>
+      <template v-if="dataerror">
+        <p v-if="flatten[0] != 'The given data was invalid.'" class="mb0">
```
Comment (djaiss) [inline resources/js/Shared/Errors.vue:14]: We need to translate this.

**Comment by djaiss** [comment]
This was such a nightmare to manage for me. I don’t master javascript so it was a pain in the arse for me.

File: resources/js/Shared/Errors.vue (hunk 1 above)
Comment (asbiin) [reply resources/js/Shared/Errors.vue:14]: I think it comes from Laravel, I don't know if it's translated

Comment (asbiin) [reply resources/js/Shared/Errors.vue:14]: Confirmed: it's not translated: https://github.com/laravel/framework/blob/8.x/src/Illuminate/Validation/ValidationException.php#L56

**Pull request merged**